app/events/admin.go

```diff
@@ -485,12 +519,16 @@ func (a *admin) callbackUnbanConfirmed(query *tbapi.CallbackQuery) error {
 func (a *admin) unban(userID int64) error {
 	if a.softBan { // soft ban, just drop restrictions
 		_, err := a.tbAPI.Request(tbapi.RestrictChatMemberConfig{
-			ChatMemberConfig: tbapi.ChatMemberConfig{UserID: userID, ChatID: a.primChatID},
+			ChatMemberConfig: tbapi.ChatMemberConfig{UserID: userID, ChatConfig: tbapi.ChatConfig{ChatID: a.primChatID}},
 			Permissions: &tbapi.ChatPermissions{
```

Comment:
Permissions list is arbitrary and always has been. I'm not sure what the correct way to deal with this. Maybe TG has a way to know "default permissions"? Btw, in this case, you dropped the Pools permission for some reason, but I can see cases where chat allows pools for users, and we may want to restore it upon unban.

Reply:
Description for polls:
```
CanSendPolls is true, if the user is allowed to send polls, implies can_send_messages
```
We don't need to care about it as we remove ability to send messages. I've left `CanSendOtherMessages` which works the same for backwards compatibility, as otherwise previously issued softbans won't restore that permission on unban.